### PR TITLE
134/pomodoro-default-times

### LINKED ISSUE #134: Feature/pom set default pomodoro and break durations.

### LINKED ISSUE #210: [PE-D] Pom with errorneous time
Timer can be set to a negative number which breaks pauses the app completely as timer does not run.

No visual feedback when the time entered for pom is not a valid number

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: helloImHai/ped#3

### LINKED ISSUE #211: [PE-D] Timer error after pom
After pom finishes, and if we input "Y", then "Y" to set a 5 minute timer, a 5 second timer is started.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: helloImHai/ped#2

### LINKED ISSUE #246: [PE-D] Negative time allowed
Pomming should not allow negative time!

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: ShengXue97/ped#6

### LINKED ISSUE #247: [PE-D] Wrong timer
The feedback says it will start 5 minute timer during the break, but in fact only 5 seconds was provided.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: ShengXue97/ped#5

### PR DESCRIPTION
**Pull request by Hardoh**
Pomodoro now heeds the data in pomodoro.json. Additional methods to change the default and rest times are available via PomodoroManager which results in the json being updated as well.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/PomCommandParser.java [hunk 1]
```diff
@@ -14,10 +14,6 @@ public PomCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TIMER);
```
Comment (BransonNg) [inline src/main/java/seedu/address/logic/parser/PomCommandParser.java:15]: Maybe add the message that time must be positive

Comment (Hardoh) [reply src/main/java/seedu/address/logic/parser/PomCommandParser.java:15]: I put that under the MESSAGE_USAGE in PomCommand hahaha. In a general invalid response.

File: src/main/java/seedu/address/logic/parser/PomCommandParser.java [hunk 2]
```diff
@@ -14,10 +14,6 @@ public PomCommand parse(String args) throws ParseException {
...
```
Comment (BransonNg) [inline src/main/java/seedu/address/logic/parser/PomCommandParser.java:14]: should throw an error if <= 0. Actually might want to raise the minimum to 0.5? (feature decision, u can decide, but at least make it <= 0, pom 0 shouldn't happen)

Comment (Hardoh) [reply src/main/java/seedu/address/logic/parser/PomCommandParser.java:14]: pom 0 as in you mean the task index?

**Review by BransonNg — APPROVED** [review]
LGTM!

Just the bit about clearer error message for invalid times and also feel like 0 should not be allowed

**Pull request merged**